generators/firechip/src/test/scala/ScalaTestSuite.scala

```diff
@@ -127,20 +116,16 @@ abstract class FireSimTestSuite(
   }
 
   clean
-  mkdirs
-  elaborate
-  generateTestSuiteMakefrags
-  generateArtefacts
   runTest("verilator", "rv64ui-p-simple", false, Seq(s"""EXTRA_SIM_ARGS=+trace-humanreadable0"""))
   //diffTracelog("rv64ui-p-simple.out")
   runSuite("verilator")(benchmarks)
-  runSuite("verilator")(FastBlockdevTests)
+  //runSuite("verilator")(FastBlockdevTests)
```

Comment:
I'm generally opposed to committing commented out code, since git preserves history it won't be hard to find this later.
I'd also suggest filing an issue to track re-enabling these tests.

Reply:
Yeah, i totally agree.